source/jormungandr/tests/check_utils.py

```diff
@@ -386,6 +386,18 @@ def query_from_str(str):
     return query
 
 
+def is_valid_feed_publisher(feed_publisher):
+    get_not_null(feed_publisher, 'id')
+    get_not_null(feed_publisher, 'name')
+    get_not_null(feed_publisher, 'license')
+    get_not_null(feed_publisher, 'url')
+
+    assert (feed_publisher["id"] == "builder")
```

Comment:
you should not test values here since it's a generic method.

those values should be tested in the unit test.